pkg/registry/core/service/allocator/utils.go

```diff
@@ -24,8 +24,8 @@ import (
 // countBits returns the number of set bits in n
 func countBits(n *big.Int) int {
 	var count int = 0
-	for _, b := range n.Bytes() {
-		count += bits.OnesCount8(uint8(b))
+	for _, w := range n.Bits() {
```

Comment:
The docs for math/big.Word say "A Word represents a single digit of a multi-precision unsigned integer" which is certainly not explaining very much.

The docs  for Bits() say "Bits is intended to support implementation of missing low-level Int functionality outside this package; it should be avoided otherwise."   I guess this counts as "missing low-level Int functionality".